manifests/conf.pp

```diff
@@ -11,10 +11,12 @@
     }
   }
 
+  $confheadertmp = epp('apt/_conf_header.epp')
+  $conftmp = epp('apt/conf.epp', { 'content' => $content })
```

Comment:
Yeah so I think I'm going to move the header call into the templates themselves, make this all a tad cleaner